workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/HyscaleGenerateServiceManifestsCommand.java

```diff
@@ -52,9 +55,9 @@
  *
  * @option appName  name of the app
  * @option serviceSpecs  list of service specs
+ * @option profiles  list of profiles for services
  *
- * Eg: hyscale generate service manifests -f s1.hspec.yaml -f s2.hspec.yaml -a sample
- *      -n dev
+ * Eg: hyscale generate service manifests -f s1.hspec.yaml -f s2.hspec.yaml -p s1.p1.yaml -a sample
```

Comment:
Instead of s1.p1.yaml it should be something like
**p1.hprof.yaml**
Service name is read from the overrides field in the profile